binding-generator/src/generator.rs

```diff
@@ -433,6 +434,11 @@ impl Generator {
 		args.push("-includeocvrs_common.hpp".into());
 		// need to have c++14 here because VS headers contain features that require it
 		args.push("-std=c++14".into());
+		// allow us to use some custom clang args
+		let clang_args = env::var_os("OPENCV_CLANG_ARGS").unwrap_or_default();
```

Comment:
I'd say that handling a `Some` value explicitly would be preferable:
```suggestion
		if let Some(clang_args) = env::var_os("OPENCV_CLANG_ARGS") {
```